### ISSUE TITLE
MetadataRpcClient.getClusterInfo always return null

### ISSUE THREAD

**Issue by stream2000** [sofastack/sofa-jraft#584]
### Describe the bug

As described in the title,  MetadataRpcClient.getClusterInfo always returns a null value in remote PD mode.

### Expected behavior

Return the actual cluster information.

### Actual behavior

Return a null.

### Steps to reproduce

Connect to remote Pd, and call this function

### Possible Cause

See GetClusterInfoResponse.

```java
// entends BaseResponse<Cluster> instead
public class GetClusterInfoResponse extends BaseResponse {

    private static final long serialVersionUID = 8811355826978037463L;

    private Cluster           cluster;

    public Cluster getCluster() {
        return cluster;
    }

    public void setCluster(Cluster cluster) {
        this.cluster = cluster;
    }

    @Override
    public String toString() {
        return "GetClusterInfoResponse{" + "cluster=" + cluster + "} " + super.toString();
    }
}
```

Maybe we should extend BaseResponse<Cluster> so that the decoder can correctly set the data in BaseReponse. And in  DefaultPlacementDriverService.handleGetClusterInfoRequest we should use respose.setData instead of response.setCluster. 

If there is no problem with my understanding, I can create a pull request to fix this bug. And please let me know if I am wrong. 

### Environment
 
- SOFAJRaft version: 1.3.6
- JVM version (e.g. `java -version`): 1.8
- OS version (e.g. `uname -a`):
- Maven version:
- IDE version:

**Comment by stream2000** [comment]
Is there any problem with this issue?  I closed it because the relative pr was merged half an hour ago. By the way, it seems like the relative pr comment was modified by one of the maintainers. It was Fix#584 at the beginning, but when the pr was merged it was Fix#537.  I thought it was a GitHub bug so I changed it back. 
 
Please let me know if there is anything I can do.

**Comment by fengjiachun** [comment]
> Is there any problem with this issue? I closed it because the relative pr was merged half an hour ago. By the way, it seems like the relative pr comment was modified by one of the maintainers. It was Fix#584 at the beginning, but when the pr was merged it was Fix#537. I thought it was a GitHub bug so I changed it back.
> 
> Please let me know if there is anything I can do.

@stream2000 I reopen this for the CLA， @seeflood

**Comment by seeflood** [comment]
> Is there any problem with this issue? I closed it because the relative pr was merged half an hour ago. By the way, it seems like the relative pr comment was modified by one of the maintainers. It was Fix#584 at the beginning, but when the pr was merged it was Fix#537. I thought it was a GitHub bug so I changed it back.
> 
> Please let me know if there is anything I can do.

Thank you for your contribution. There is no problem with this issue. We reopened the issue because the PR you submitted was merged, but the cla was not signed. I just checked the cla database and found that there is no signature data for your cla. Could you please re-sign cla?
Open http://cla.sofastack.tech/ and click "Sign Individual CLA".If it succeeds, a prompt will appear:

![image](https://user-images.githubusercontent.com/26001097/117913110-57940100-b313-11eb-9cb9-f3623520f7b9.png)

Sorry for the inconvenience

**Comment by stream2000** [comment]
> > Is there any problem with this issue? I closed it because the relative pr was merged half an hour ago. By the way, it seems like the relative pr comment was modified by one of the maintainers. It was Fix#584 at the beginning, but when the pr was merged it was Fix#537. I thought it was a GitHub bug so I changed it back.
> > Please let me know if there is anything I can do.
> 
> @stream2000
> Thank you for your contribution. There is no problem with this issue. We reopened the issue because the PR you submitted was merged, but the cla was not signed. I just checked the cla database and found that there is no signature data for your cla.
> Could you please re-sign cla?
> Open http://cla.sofastack.tech/ and click "Sign Individual CLA".If it succeeds, a prompt will appear:
> ![image](https://user-images.githubusercontent.com/26001097/117913297-b78aa780-b313-11eb-92ae-f79774d16c05.png)
> 
> ![image](https://user-images.githubusercontent.com/26001097/117913110-57940100-b313-11eb-9cb9-f3623520f7b9.png)
> 
> Maybe you just authorized to access the cla website through github oauth last time, but you did not click the button to sign cla after you authorized oauth?
> Sorry for the inconvenience.

I have submitted the request multiple times and the prompt appeared eventually.. Maybe we can improve the UI so that it can tell us whether the submission is failed or not😺

**Comment by seeflood** [comment]
> > > Is there any problem with this issue? I closed it because the relative pr was merged half an hour ago. By the way, it seems like the relative pr comment was modified by one of the maintainers. It was Fix#584 at the beginning, but when the pr was merged it was Fix#537. I thought it was a GitHub bug so I changed it back.
> > > Please let me know if there is anything I can do.
> > 
> > 
> > @stream2000
> > Thank you for your contribution. There is no problem with this issue. We reopened the issue because the PR you submitted was merged, but the cla was not signed. I just checked the cla database and found that there is no signature data for your cla.
> > Could you please re-sign cla?
> > Open http://cla.sofastack.tech/ and click "Sign Individual CLA".If it succeeds, a prompt will appear:
> > ![image](https://user-images.githubusercontent.com/26001097/117913297-b78aa780-b313-11eb-92ae-f79774d16c05.png)
> > ![image](https://user-images.githubusercontent.com/26001097/117913110-57940100-b313-11eb-9cb9-f3623520f7b9.png)
> > Maybe you just authorized to access the cla website through github oauth last time, but you did not click the button to sign cla after you authorized oauth?
> > Sorry for the inconvenience.
> 
> I have submitted the request multiple times and the prompt appeared eventually.. Maybe we can improve the UI so that it can tell us whether the submission is failed or not😺

Thanks!
The oauth protocol is too slow because of gfw  :(
Maybe I can add a loading flag in UI,and tell user that they failed signing cause the well-known reason

**Comment by fengjiachun** [comment]
tks @stream2000
